### PR TITLE
docs: updated windows dev environment git configuration settings for …

### PR DESCRIPTION
**Pull request by HeikkM**
Updated README.md with instructions how to set git configs when developing in windows environment. Symbolic links behave differently than in Linux and will cause building errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,16 @@ They are based on the [npm-check-updates](https://github.com/raineorshine/npm-ch
 > having duplicates in the yarn.lock, you can run `yarn dedupe --check` and `yarn dedupe` to
 > apply deduplication. The duplicate check is enforced in the example github actions.
 
+### Developing on Windowns
+
+Monorepo uses symbolic links and you should check your git configurations when operating on Windows.
+If git config has core.symlinks key set to false when cloning the repository, building an app will result : 'Error: Export encountered errors on following paths:'. Note that `yarn dev` with apps could give the error, but still will get up and running. Also if your `/packages/common-i18n/src/locales` does not have folder `default` and has a file instead, you have a problem.
+
+Set your git configs before cloning the repository.
+Check our git config in console with `git config --list`. You should have either global or system key `core.symlinks` set to true.
```
Comment (kristianjokela) [inline README.md:194]: Check our git -> Check your git

File: README.md [hunk 2]
```diff
@@ -185,6 +185,16 @@ They are based on the [npm-check-updates](https://github.com/raineorshine/npm-ch
 > having duplicates in the yarn.lock, you can run `yarn dedupe --check` and `yarn dedupe` to
 > apply deduplication. The duplicate check is enforced in the example github actions.
 
+### Developing on Windowns
+
+Monorepo uses symbolic links and you should check your git configurations when operating on Windows.
+If git config has core.symlinks key set to false when cloning the repository, building an app will result : 'Error: Export encountered errors on following paths:'. Note that `yarn dev` with apps could give the error, but still will get up and running. Also if your `/packages/common-i18n/src/locales` does not have folder `default` and has a file instead, you have a problem.
```
Comment (kristianjokela) [inline README.md:191]: You could format core.symlinks as code too, ie. `core.symlinks`

**Review by kristianjokela — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -185,6 +185,16 @@ They are based on the [npm-check-updates](https://github.com/raineorshine/npm-ch
 > having duplicates in the yarn.lock, you can run `yarn dedupe --check` and `yarn dedupe` to
 > apply deduplication. The duplicate check is enforced in the example github actions.
 
+### Developing on Windowns
```
Comment (karisal-anders) [inline README.md:188]: typo: Windowns -> Windows

**Comment by karisal-anders** [comment]
PR #262 was merged, could this be closed now?

**Comment by karisal-anders** [comment]
As this has been open for over a month now and PR #262 replaced this and was merged I will close this to clean up the PRs.

**Pull request closed without merging**